### PR TITLE
Added environments

### PR DESCRIPTION
**Pull request by Froxz**
Added support for environments #1102

### CODE REVIEW

**Review by Nyholm — APPROVED** [review]
Lovely. Thank you for adding docs and tests.

**Comment by Nyholm** [comment]
It seams like the tests are still failing

File: lib/Github/Api/Environment.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+namespace Github\Api;
+
+/**
+ * Listing, creating and updating environments.
+ *
+ * @link https://docs.github.com/en/rest/deployments/environments?apiVersion=2022-11-28#
+ */
+class Environment extends AbstractApi
+{
+    use AcceptHeaderTrait;
```
Comment (acrobat) [inline lib/Github/Api/Environment.php:12]: This api doesn't require a custom beta accept header, so the trait usage can be removed

**Review by acrobat — CHANGES_REQUESTED** [review]

**Comment by acrobat** [comment]
The failing BC checker workflow can be ignored as it is not related to this change. One smal change/comment from my side, otherwise go to go!

File: lib/Github/Api/Environment.php (hunk 1 above)
Comment (Froxz) [reply lib/Github/Api/Environment.php]: Hi @acrobat thank you, is now removed.

**Comment by Nyholm** [comment]
Thank you

**Comment by acrobat** [comment]
Thanks @Froxz! And congrats on your first contribution! 🎉

**Pull request merged**